e2e-tests/upgrade-consistency-sharded-tls/compare/service_some-name-rs0-1150.yml

```diff
@@ -15,7 +15,8 @@ metadata:
       name: some-name
 spec:
   ports:
-    - name: mongodb
+    - appProtocol: mongo
+      name: mongodb
```

Comment:
we shouldn't have this in `1.15.0`

Reply:
@sorenmat please don't forget about this. we'll start code freeze starting from tomorrow